bip-0085.mediawiki

```diff
@@ -2,7 +2,7 @@
   BIP: 85
   Layer: Applications
   Title: Deterministic Entropy From BIP32 Keychains
-  Author: Ethan Kosakovsky <ethankosakovsky@protonmail.com>
+  Author: Aneesh Karve <dowsing.seaport0d@icloud.com>
```

Comment:
```suggestion
  Author: Ethan Kosakovsky <ethankosakovsky@protonmail.com>, Aneesh Karve <dowsing.seaport0d@icloud.com>
```

Reply:
I tried the first one [earlier](https://github.com/bitcoin/bips/pull/1600/commits/fdaf02c3f77bf98363bb6045319f29b9ffd86281) and it made CI cranky. Let's try the second one.